src/services/doppler-api-client.ts

```diff
@@ -80,4 +129,41 @@ export class HttpDopplerApiClient implements DopplerApiClient {
       return { success: false, error: error };
     }
   }
+
+  public async getSubscriber(email: string): Promise<ResultWithoutExpectedErrors<Subscriber>> {
+    try {
+      const { jwtToken, userAccount } = this.getDopplerApiConnectionData();
+
+      const response = await this.axios.request({
+        method: 'GET',
+        url: `/accounts/${userAccount}/subscribers/${email}`,
+        headers: { Authorization: `token ${jwtToken}` },
+      });
+
+      const subscriber = {
```

Comment:
Wouldn't it be best to use spread operator here? like this
```
const subscriber = { ...response.data};
```
And then map links and fields apart.